bundles/org.openhab.binding.doorbird/src/main/java/org/openhab/binding/doorbird/internal/action/DoorbirdActions.java

```diff
@@ -54,8 +54,10 @@ public void setThingHandler(@Nullable ThingHandler handler) {
     @RuleAction(label = "restart the Doorbird", description = "Restarts the Doorbird device.")
     public void restart() {
         logger.debug("Doorbird action 'restart' called");
+        DoorbellHandler handler = this.handler;
         if (handler != null) {
             handler.actionRestart();
+            this.handler = handler;
```

Comment:
Why setting again this.handler with handler ?

Reply:
Fixed